src/qt/rpcconsole.cpp

```diff
@@ -1197,6 +1196,7 @@ void RPCConsole::updateDetailWidget()
     // This check fails for example if the lock was busy and
     // nodeStateStats couldn't be fetched.
     if (stats->fNodeStateStatsAvailable) {
+        ui->peerServices->setText(GUIUtil::formatServicesStr(stats->nodeStateStats.their_services));
```

Comment:
Tested the result when the stats aren't available, which looks good:

![Screenshot from 2022-07-13 14-07-22](https://user-images.githubusercontent.com/6399679/178730160-c0daf9f4-5f34-4039-a370-be4e4fe055e2.png)